### ISSUE TITLE
[BUG] Very small metric values are displayed incorrectly in the run list

### ISSUE THREAD

**Issue by iver56** [mlflow/mlflow#1901]
### System information
- **Have I written custom code (as opposed to using a stock example script provided in MLflow)**: See section "Code to reproduce issue" below
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: MLflow 1.3 Tracking UI hosted on [Mflux.ai](https://www.mflux.ai) with Linux Ubuntu 18.04.
- **MLflow installed from (source or binary)**: pip (PyPI)
- **MLflow version (run ``mlflow --version``)**: 1.3
- **Python version**: The remote MLflow UI runs on Python 3.7, my local MLflow client runs Python 3.6
- **npm version, if running the dev UI**: N/A
- **Exact command to reproduce**: N/A

### Describe the problem

The displayed value in the list looks like a relatively large value (close to 1.1), but the value is actually very close to zero. See the following annotated screenshot:

![bilde](https://user-images.githubusercontent.com/1470603/66111462-390f4500-e5c9-11e9-876a-d2ceddb80c1f.png)

I would expect this value to be displayed like 0.0000000..., or if not, at least include the part that says "e-13"

### Code to reproduce issue

```
import mlflow

mlflow.log_metric("final_error", 1.11910480882216e-13)
```

### Other info / logs
N/A

**Comment by iver56** [comment]
This issue still persists, and should not be closed

**Comment by harupy** [comment]
The cause is that the maximum width of `span` tag containing metric values is restricted to `120px`. So a metric value with many digits such as `1.11910480882216e-13` gets truncated and displayed incorrectly.

```javascript
const styles = {
  metricParamCellContent: {
    display: "inline-block",
    maxWidth: 120,
  },
};
```

https://github.com/mlflow/mlflow/blob/master/mlflow/server/js/src/components/BaggedCell.js#L7-L12


```javascript
            ...
            <span
              className="metric-param-value run-table-container"
              style={styles.metricParamCellContent}
            >
              {value}
            </span>
            ...
```

https://github.com/mlflow/mlflow/blob/master/mlflow/server/js/src/components/BaggedCell.js#L76--L81

**Comment by iver56** [comment]
Stalebot, please keep this issue open

**Comment by iver56** [comment]
Please unmark as stale

**Comment by iver56** [comment]
Please unmark as stale

**Comment by AveshCSingh** [comment]
Closing as this was fixed by #2022
